src/client/Client.cc

```diff
@@ -8558,6 +8558,10 @@ loff_t Client::_lseek(Fh *f, loff_t offset, int whence)
     ceph_abort();
   }
 
+  if (f->pos < 0){
```

Comment:
Missing space between ")" and "{"

Reply:
we should check the result offset in lseek, return -EINVAL if it's negtive value